BUILD.md

```diff
@@ -295,3 +298,78 @@ Module names can be found [here](bin/run_lint#L205) in the codebase.
   then you need to set `LD_LIBRARY_PATH` with:
   `source /opt/intel/sgxsdk/environment` .
   For details, see [PREREQUISITES](PREREQUISITES.md)
+
+# <a name="avaloncentos8"></a>Run Avalon on CentOS 8.2
+## <a name="sgxinstall"></a>SGX driver and PSW package installation
+1. Install SGX drivers
+   ```
+   sudo yum update
+   sudo yum install kernel-devel gcc
+   wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/centos8.1-server/sgx_linux_x64_driver_2.6.0_602374c.bin
+   chmod +x sgx_linux_x64_driver_2.6.0_602374c.bin
+   sudo ./sgx_linux_x64_driver_2.6.0_602374c.bin
+   
+   You should be able to see /dev/isgx device mapping on your host.
+   ```
+2. Install SGX PSW package
+   ```
+   wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/centos8.1-server/sgx_rpm_local_repo.tgz
+   tar -xvfsgx_rpm_local_repo.tgz
+   cd sgx_rpm_local_repo/
+   sudo yum localinstall -y \
+     libsgx-ae-epid-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-ae-le-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-ae-pce-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-ae-qe3-1.7.100.2-1.el8.x86_64.rpm \
+     libsgx-ae-qve-1.7.100.2-1.el8.x86_64.rpm \
+     libsgx-aesm-epid-plugin-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-aesm-launch-plugin-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-aesm-pce-plugin-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-aesm-quote-ex-plugin-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-enclave-common-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-enclave-common-devel-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-epid-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-epid-devel-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-launch-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-launch-devel-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-pce-logic-1.7.100.2-1.el8.x86_64.rpm \
+     libsgx-qe3-logic-1.7.100.2-1.el8.x86_64.rpm \
+     libsgx-quote-ex-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-quote-ex-devel-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-uae-service-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-urts-2.10.100.2-1.el8.x86_64.rpm \
+     sgx-aesm-service-2.10.100.2-1.el8.x86_64.rpm \
+     libsgx-aesm-ecdsa-plugin-2.10.100.2-1.el8.x86_64.rpm
```

Comment:
For CentOS 8.1 SGX driver and SDK installation (version 2.10) , can we follow the steps in https://download.01.org/intel-sgx/sgx-linux/2.10/docs/Intel_SGX_Installation_Guide_Linux_2.10_Open_Source.pdf
See page 10. We don’t need to each install rpm packages separately.

Reply:
Okay. I saw that earlier. But I want to exclude installation of packages related to DCAP if I am only interested in EPID based attestation. Let me check if this can be done.

Reply:
Done